packages/query/src/stacks/stacks-client.ts

```diff
@@ -87,6 +91,36 @@ export function stacksClient(basePath: string) {
       );
       return resp.data;
     },
+    async getEventsForAddressTransaction(address: string, txId: string, signal: AbortSignal) {
```

Comment:
I added these as potential replacements to `getAccountTransactionsWithTransfers` but haven't spent long enough figuring out how to use them yet